specs/gluster_object_detail_views.adoc

```diff
@@ -0,0 +1,198 @@
+// vim: tw=79
+:imagesdir: ./images/
+
+= Tendrl Definitions: Display list of volume and host associated with a gluster cluster discovered by Tendrl
+
+== Problem description
+
+Display list of volume and host associated with a gluster cluster present in Tendrl. 
+
+*Host List*
+
+The details shown for each host are -
+
+* Status
+* Name
+* Role
+* Alerts
+
+The host list should have functionality for 
+
+* filtering by name, role and status.
```

Comment:
In case of gluster all the nodes have same role. But have he filtering mechanism in place which would be helpful later.

Reply:
I am ok with `gluster/peer`